ubuntu/common/hpc-tuning.sh

```diff
@@ -30,7 +30,37 @@ cat << EOF >> /etc/security/limits.conf
 EOF
 
 echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh1 = 4096" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh2 = 8192" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh3 = 16384" >> /etc/sysctl.conf
+echo "sunrpc.tcp_max_slot_table_entries = 128" >> /etc/sysctl.conf
+
+
+## Systemd service for starting sunrpc and adding setting parameters
+cat <<EOF >/usr/sbin/sunrpc_tcp_settings.sh
+#!/bin/bash
+
+modprobe sunrpc
 sysctl -p
+EOF
+chmod 755 /usr/sbin/sunrpc_tcp_settings.sh
+
+cat <<EOF >/etc/systemd/system/sunrpc_tcp_settings.service
+[Unit]
+Description=Set sunrpc tcp settings
+
+[Service]
+User=root
+Type=oneshot
+ExecStart=/usr/sbin/sunrpc_tcp_settings.sh
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable sunrpc_tcp_settings
+systemctl start sunrpc_tcp_settings
+
 
 # Install WALinuxAgent
 apt-get install python3-setuptools
```

Comment:
Why is this done here rather than utils file?

Reply:
This refers to the install of python3-setuptools? That's a good point we probably should move that.

Reply:
Prefer separate PR, as it's not related to the major change in this PR.